database/index.js

```diff
@@ -8,65 +8,47 @@ let connection = mysql.createConnection ({
 
 const addRelatedArtist = function(artistId, relatedArtistId, callback) {
   // TODO: add error handling
-  const query = `INSERT INTO relatedArtists (main_Artist_ID, related_Artist_ID) VALUES (${connection.escape(artistId)}, ${connection.escape(relatedArtistId)})`;
+  const query = `INSERT INTO related_artists (artist_id, related_artist_id) VALUES (${connection.escape(artistId)}, ${connection.escape(relatedArtistId)})`;
   connection.query(query, (err, data) => {
-    if (err) {
-      console.log('db query error: ', err);
-      callback(err, null);
-    } else {
-      console.log('db query success');
-      callback(null, data);
-    }
+    if (err) return callback(err, null);
```

Comment:
Just want to confirm that you want callback(err, null).....
since it will be sending back err to the client. not sure if you want that?

Reply:
Yup, I want the DB functions to always invoke the callback with error first if the database responds with error. For now, I will also send those errors to the client, but may change that in the future